include/deal.II/matrix_free/matrix_free.h

```diff
@@ -381,7 +382,14 @@ class MatrixFree : public Subscriptor
      * to work on a level, its dofs must be distributed by using
      * <code>dof_handler.distribute_mg_dofs(fe);</code>.
      */
-    unsigned int level_mg_handler;
+    unsigned int mg_level;
+
+    /**
+     * Alias for mg_level
+     *
+     * @deprecated Use mg_level instead.
+     */
+    DEAL_II_DEPRECATED unsigned int &level_mg_handler;
```

Comment:
excellent, that is a smart way to do it. :+1:

Reply:
Smart yes, but the reference variable did remove the implicit copy constructor of `MatrixFree::AdditionalData` that was previously available. I think we should introduce an explicit copy operator to restore backward compatibility.

Reply:
I'll do!